package.json

```diff
@@ -75,6 +75,7 @@
     "@typescript-eslint/eslint-plugin": "^3.9.0",
     "@typescript-eslint/parser": "^3.9.0",
     "@unstoppabledomains/sizecheck": "^4.0.0",
+    "@zilliqa-js/core": "^3.0.0",
```

Comment:
@sudoryan Since we need this for tests-only please move this dependency to the dev section.